pgxn/neon/file_cache.c

```diff
@@ -317,17 +337,6 @@ lfc_init(void)
 							NULL,
 							NULL);
 
-	DefineCustomStringVariable("neon.file_cache_path",
-							   "Path to local file cache (can be raw device)",
-							   NULL,
-							   &lfc_path,
-							   "file.cache",
-							   PGC_POSTMASTER,
-							   0,
-							   NULL,
-							   NULL,
-							   NULL);
-
```

Comment:
Something to think about: if we remove this custom GUC variable, will that cause errors because the control plane continues trying to set it?

Do we need to keep this around (but unused) until the control plane is updated to no longer set this value?

Reply:
Restored